css/css-grid/WEB_FEATURES.yml

```diff
@@ -0,0 +1,3 @@
+features:
+- name: grid
```

Comment:
I can introduce logic so that when looking at all of the tests in grid and below:

- Given we are analyzing the feature in  a YML file with "**" and as we traverse the (nested) sub-directories
- if we encounter a YML file in a sub-directory with IGNORE_PARENT,
- exclude the tests in that sub-directory (and any nested sub-directories of that sub-directory)

Reply:
Thanks, that sounds good! I don't think we should have cases where a test belongs both to a feature and its subfeature.

Reply:
> I don't think we should have cases where a test belongs both to a feature and its subfeature.

@foolip, ahh that's good to know. In that case, we could remove the apply_mode field completely. Then we could use the presence of a child YML file to tell it to ignore the parent.

Reply:
Yes, remove it. If we ever need to do it. We can add a field like "APPLY_TO_PARENT" if we ever need to bring it back. I'll make a note about that in the RFC

Reply:
Thanks! I've removed them now and will merge this.